lightning/src/routing/network_graph.rs

```diff
@@ -488,6 +503,13 @@ impl std::fmt::Display for NetworkGraph {
 	}
 }
 
+/// By default, just do nothing when scoring channels. A user is free to extend these
+/// definitions to meet their needs.
```

Comment:
It would be great to have example in documentation, like a type Metadata being a struct BasicReliability { uptime: u32, flapiness: u32 }. Just a doc example not a default implementation.